### ISSUE TITLE
Boxmenu duration label and progress should appear before the view button

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2119]
### Subject of the issue
The button should be the last readable element

**Comment by oliverfoster** [comment]
merged
